pypeit/spectrographs/vlt_xshooter.py

```diff
@@ -695,6 +698,8 @@ def default_pypeit_par(self):
 
         # 1D wavelength solution
         par['calibrations']['wavelengths']['lamps'] = ['ThAr_XSHOOTER_UVB']
+        # TODO: This is a KLUDGE; default_pypeit_par should be a
```

Comment:
@jhennawi, @EmAstro: Is there a reason that `nonlinear_counts` needs to be a parameter?  I'm not crazy about having most of the `default_pypeit_par` methods being static with the exception of this one.  I think it's okay if this means we need to change the definition of the rest of the spectrographs, but we need to check.

Reply:
Yeah this is deprecated now. We synthesize it from the detector parameters in the base classes. I'm fine if it is removed in a future PR.